pom.xml

```diff
@@ -23,7 +23,7 @@
   <parent>
     <groupId>org.apache.apex</groupId>
     <artifactId>malhar</artifactId>
-    <version>3.5.0</version>
+    <version>3.6.0</version>
```

Comment:
Why is it necessary to upgrade malhar version? Is there any particular issue fixed in malhar 3.6.0 that justifies the upgrade?